### DISCUSSION TITLE
Docker镜像地址在哪里

### DISCUSSION THREAD

**Discussion by vbem** [MetaCubeX/mihomo#784] [Q&A] [answered]
看了 https://github.com/MetaCubeX/Clash.Meta/blob/Alpha/.github/workflows/build.yml 没看出了镜像上传到docker.io哪里了，请告知下谢谢~

**Comment by ZGLinus** [comment, marked as answer]
https://hub.docker.com/r/metacubex/clash-meta

**Reply by vbem** [reply]
thanks
